src/dkg_hybriddkg.erl

```diff
@@ -272,51 +285,52 @@ handle_msg(DKG, _Sender, timeout) ->
 %%          else
 %%              delay ← delay(T )
 %%              start timer(delay)
-handle_msg(DKG=#dkg{leader=Leader, t=T, n=N, f=F, qhat=Qhat0, rhat=Rhat0, l_next=LNext},
+handle_msg(DKG = #dkg{leader = Leader, t=T, n=N, f=F, shares_seen=SharesSeen0,
+                      ready_proofs=ReadyProofs0, leader_cap=LeaderCap},
            Sender,
-           {signed_leader_change, Lbar, Q, RorM}=LeaderChangeMsg) when Lbar > Leader ->
+           {signed_leader_change, ProposedLeader, Q, RorM}=LeaderChangeMsg) when ProposedLeader > Leader ->
```

Comment:
Maybe change the `Q` and `RorM` here as well? So `Q` -> `Shares` and `RorM` -> `EitherProof` or `Proofs`?

Reply:
good catch, will do